synapse/replication/tcp/protocol.py

```diff
@@ -250,8 +248,23 @@ def lineReceived(self, line):
         Args:
             cmd: received command
         """
-        handler = getattr(self, "on_%s" % (cmd.NAME,))
-        await handler(cmd)
+        handled = False
+
+        # First call any command handlers on this instance. These are for TCP
+        # specific handling.
```

Comment:
there are some words in the docstring for both this method and the `BaseReplicationStreamProtocol` which are now overly simplisiitc.